app/services/get-export-csv.js

```diff
@@ -89,6 +89,13 @@ var getCsv = function (organisationLevel, result, tab, fields, fieldNames) {
       }
       break
     case tabs.OVERVIEW:
+      if (Array.isArray(result.overviewDetails)) {
```

Comment:
Is this an Array for all levels above offender manager? If so, maybe do the `if` on organisational level to match other `if`s in this file